### ISSUE TITLE
[Multi Combobox]: Accessibility issue when the readonly enabled for multi combo box.

### ISSUE THREAD

**Issue by achuth2912** [UI5/webcomponents#12705]
### Bug Description

On the multi combo box select 3-4 items and enable the readOnly flag.
When the user reached this component using the Screen reader it announces "select tags combobox collapsed" which means users will think combobox is still interactive and the read only state is not  being announced.

### Affected Component

Multi Combobox

### Expected Behaviour

The Screen reader must announce as the component is non-interactive / its in readonly mode.

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/docs/inputs-multicombobox--docs

### Steps to Reproduce

1. Select 3-4 items form the combo box.
2. Enable the ReadOnly Flag.
3. Then navigate to the component with Screen reader
...

### Priority

None

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Operating System

Windows

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @UI5/ui5-team-ctr-ril ,

I've tested the use case of the user in UI5 WebC and "readonly" is not announced as reported. OpenUI5 (with editable "false")  announces it as it should.

Could you please check investigate further?

Regards,
Plamen Ivanov

**Comment by achuth2912** [comment]
Any Update on this Issue ?
We have a customer Incident waiting on this one

**Comment by MapTo0** [comment]
Hello!

UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of browser and screen reader. UI5 Web Components focus on compliance with the standards, rely on automated checks for accessibility and manual tests with reference testing environments.

The mentioned behaviour is confirmed gap in JAWS.
Please check 

- https://github.com/FreedomScientific/standards-support/issues/509
- https://github.com/FreedomScientific/standards-support/issues/804

I am closing the issue as we can not fix this on our side.

Best Regards!
Martin
